### PR TITLE
Added first pass at release notes for 0.12.5

### PR DESCRIPTION
**Pull request by alexwykoff**
Auditors: @bbondy

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,19 @@
...
+- Fixed the ability to change default search via preferences ([#4695](https://github.com/brave/browser-laptop/issues/4695))([#4696](https://github.com/brave/browser-laptop/issues/4696))
+- Fixed Brave Wallet showing coinbase as an option for unsupported countries ([#4615](https://github.com/brave/browser-laptop/issues/4615))
+- Fixed double clicking bookmark in bookmark manager opening many many tabs ([#4523](https://github.com/brave/browser-laptop/issues/4523))
+- Fixed toggling Bookmarks Toolbar from "Bookmarks" menu ([#3356](https://github.com/brave/browser-laptop/issues/3356))
+- Fixed bookmarks toolbar "hover after click" behavior on folders ([#4361](https://github.com/brave/browser-laptop/issues/4361))
+- Fixed an edge case of multiple Payment History receipts being generated on the same day ([#4632](https://github.com/brave/browser-laptop/issues/4632))([#4605](https://github.com/brave/browser-laptop/issues/4605))
+- Fixed Windows titlebar to ignore right alt key (AltGr) ([#4628](https://github.com/brave/browser-laptop/issues/4628))([#4626](https://github.com/brave/browser-laptop/issues/4626))
 
 ## [0.12.4](https://github.com/brave/browser-laptop/releases/v0.12.4dev)
```
Comment (bsclifton) [inline CHANGELOG.md]: Some of the links seemed to be wrong (they went to a PR instead of the issue, sometimes not the right PR either)

Comment (alexwykoff) [inline CHANGELOG.md]: We do leave PRs in here as well. I did notice at least one bad link while testing (transposed two numbers while editing). 
I'll give this a thorough scrub and resubmit. 
Thanks for the review!

**Comment by bbondy** [comment]
++

**Comment by bbondy** [comment]
btw pls also tag @luixxiul on these reviews, as he's good at spotting things in release notes.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,19 @@
...
+- Added opt-out for crash reporting ([#4479](https://github.com/brave/browser-laptop/issues/4479))([#4606](https://github.com/brave/browser-laptop/issues/4606))
+- Added numerous UI style changes ([#4627](https://github.com/brave/browser-laptop/issues/4627))([#4624](https://github.com/brave/browser-laptop/issues/4624))([#4566](https://github.com/brave/browser-laptop/issues/4566))([#4603](https://github.com/brave/browser-laptop/issues/4603))([#4562](https://github.com/brave/browser-laptop/issues/4562))([#4616](https://github.com/brave/browser-laptop/issues/4616))([#4537](https://github.com/brave/browser-laptop/issues/4537))([#1725](https://github.com/brave/browser-laptop/issues/1725))([#4614](https://github.com/brave/browser-laptop/issues/4614))([#4658](https://github.com/brave/browser-laptop/issues/4658))([#4636](https://github.com/brave/browser-laptop/issues/4636))([#4292](https://github.com/brave/browser-laptop/issues/4292))
+- Added the ability to disable spellcheck for custom filters list ([#4637](https://github.com/brave/browser-laptop/issues/4637))([#4619](https://github.com/brave/browser-laptop/issues/4619))
```
Comment (bbondy) [inline CHANGELOG.md]: This one is just a bugfix and not an option. 
And one of those issues links to the PR, better to just remove that. No duplicates listed either, leave those out if there are any.

**Pull request merged**